### PR TITLE
Crkbd/thunderbird2086

### PR DESCRIPTION
**Pull request by Thunderbird2086**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/crkbd/keymaps/thunderbird2086/config.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+#define RGBLIGHT_SLEEP
+
+#ifdef RGBLIGHT_ENABLE
+#    define RGBLIGHT_LIMIT_VAL 150
+#    define RGBLIGHT_HUE_STEP 16
+#    define RGBLIGHT_SAT_STEP 32
+#    define RGBLIGHT_VAL_STEP 32
+
+#    define RGBBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/crkbd/keymaps/thunderbird2086/config.h:62]:
```suggestion
```

File: keyboards/crkbd/keymaps/thunderbird2086/keycodes.h [hunk 2]
```diff
@@ -0,0 +1,66 @@
+#pragma once
```
Comment (drashna) [inline keyboards/crkbd/keymaps/thunderbird2086/keycodes.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/crkbd/keymaps/thunderbird2086/keymap.c [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#include QMK_KEYBOARD_H
+#include "layers_block.h"
+
```
Comment (drashna) [inline keyboards/crkbd/keymaps/thunderbird2086/keymap.c:3]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/crkbd/keymaps/thunderbird2086/layers_block.h [hunk 4]
```diff
@@ -0,0 +1,86 @@
+#pragma once
```
Comment (drashna) [inline keyboards/crkbd/keymaps/thunderbird2086/layers_block.h:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/crkbd/keymaps/thunderbird2086/oled.c [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+
+#if defined(RGB_MATRIX_ENABLE) || defined(RGBLIGHT_ENABLE)
+
+#if defined(RGBLIGHT_ENABLE)
+extern rgblight_config_t rgblight_config;
+#endif
+
+#if defined(RGB_MATRIX_ENABLE)
+extern rgb_config_t rgb_matrix_config;
+#endif
```
Comment (drashna) [inline keyboards/crkbd/keymaps/thunderbird2086/oled.c:90]: There are functions for getting the values for these, and these shouldn't be used directly. 

https://docs.qmk.fm/#/feature_rgblight?id=query
https://docs.qmk.fm/#/feature_rgb_matrix?id=query-current-status

**Review by drashna — APPROVED** [review]

File: keyboards/crkbd/keymaps/thunderbird2086/rules.mk [hunk 6]
```diff
@@ -0,0 +1,18 @@
+MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/thunderbird2086/rules.mk:1]:
```suggestion
MOUSEKEY_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by Thunderbird2086** [comment]
@fauxpark could you review again?

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**